### ISSUE TITLE
"mesPreset" Not optional

### ISSUE THREAD

**Issue by michaelwoodew** [Azure-Samples/media-services-dotnet-functions-integration#21]
`"mesPreset" : "Adaptive Streaming",         // Optional but required to encode with Media Encoder Standard (MES). If MESPreset contains an extension "H264 Multiple Bitrate 720p with thumbnail.json" then it loads this file from ..\Presets`

I'm only intending to use indexer for mp3 files and don't require re-encoding. if "mesPreset" is not defined the job fails.

**Comment by xpouyat** [comment]
Hello,
I tested with an MP3 file and I have no issue. Job is a success.

You should try with this body when calling submit-job:

 "body": {
                        "assetId": "@{body('create-empty-asset')['assetId']}",
                        "indexV2Language": "EnUs"
                    },

Are you using indexer v1 or v2 ? How did you create the asset ?

**Comment by michaelwoodew** [comment]
You are correct. It did work. However, I did have to change the preset path in the `submit_task` function and the `jobHelpers.csx` file.

**from** `presetPath = Path.Combine(homePath, @"site\repository\media-functions-for-logic-app\presets\" + presetfilename);`

**to** `presetPath = Path.Combine(homePath, @"site\wwwroot\presets\" + presetfilename);`

**Comment by xpouyat** [comment]
I see. It's because you did not deployed the functions with the ARM template using the Project setting.... With the project setting, the folder hierarchy is "site\repository\media-functions-for-logic-app\presets\"

**Comment by michaelwoodew** [comment]
I deployed from the button in the readme after I forked,

**Comment by xpouyat** [comment]
Did you create a preset and deployed in in Presets ?
There are two Presets folders in the repo:
\Presets
And
\media-functions-for-logic-app\presets

The functions from 'media-functions-for-logic-app' use the \media-functions-for-logic-app\presets folder.

**Comment by michaelwoodew** [comment]
I'm using the presets that came with the sample code.
I first used the media services deploy button  here: https://github.com/Azure-Samples/media-services-dotnet-functions-integration

and then deployed the function app with the deploy button here https://github.com/Azure-Samples/media-services-dotnet-functions-integration/tree/master/media-functions-for-logic-app

using 'Second Logic App : using Azure Storage trigger'

This method places the presets folder in "site > wwwroot"
Directly inside that route in the function folders "add-textfile-to-asset", "check-copy-blob-to-asset-status"
and so on.

Small side note.
is there a reason "itamybwd2fwne" is added to the services created using the deploy button?
for example if i call the storage container "IndexerStorage" during deployment the actually name becomes "IndexerStorageitamybwd2fwne".

This string was added to the App service plan (twice itamybwd2fwneitamybwd2fwne), the Functions App Service, the storage account and media services.
